Plots/Scatter/NCL_scatter_3.py

```diff
@@ -0,0 +1,98 @@
+"""
+NCL_scatter_3.py
+================
+This script illustrates the following concepts:
+    - Drawing a scatter plot over a map
+    - Drawing markers on a map
```

Comment:
Do the second and third comments mean the same thing?

Reply:
I revised the "concepts" and made them a bit more comprehendible and applicable to the code